source/ambassador_host_test.go

```diff
@@ -141,3 +658,77 @@ func TestParseAmbLoadBalancerService(t *testing.T) {
 		}
 	}
 }
+
+func convertAmbassadorHostToUnstructured(hp *ambassador.Host, s *runtime.Scheme) (*unstructured.Unstructured, error) {
+	unstructuredAmbassadorHost := &unstructured.Unstructured{}
+	if err := s.Convert(hp, unstructuredAmbassadorHost, context.Background()); err != nil {
+		return nil, err
+	}
+	return unstructuredAmbassadorHost, nil
+}
+
+func newAmbassadorDynamicKubernetesClient() (*fakeDynamic.FakeDynamicClient, *runtime.Scheme) {
```

Comment:
If one is not careful and look at the return values, one may be mislead into thinking this creates a non-fake client. Perhaps rename?